### ISSUE TITLE
Problem building the Windows installer

### ISSUE THREAD

**Issue by gagnonpl** [podonoghue/usbdm-eclipse-makefiles-build#19]
Hi @podonoghue, I would have sent this as a private msg but I couldn't find a public email address.

I successfully built the project on Windows and now I'm trying to build the msi installer.   If I go to `/PackageFiles/Wix` and run `Make_Installer.cmd`, heat.exe complains that the `PackageFiles/UpdateSite` dir does not exist.  Indeed it doesn't in the src dir by default, so I'm thinking maybe it's generated during the build process.  Could you tell me how to get that dir?

Also may be unrelated, but there are more errors later while the script runs:
> <path-to-repo>\usbdm-eclipse-makefiles-build\PackageFiles\Wix\usbdm.wxs(193) : error LGHT0094 : Unresolved reference to symbol 'WixComponentGroup:Cg.WizardPatchData' in section 'Product:*'.
> <path-to-repo>\usbdm-eclipse-makefiles-build\PackageFiles\Wix\usbdm.wxs(308) : error LGHT0094 : Unresolved reference to symbol 'WixComponentGroup:Cg.i386_Bin' in section 'Product:*'.
> <path-to-repo>\usbdm-eclipse-makefiles-build\PackageFiles\Wix\usbdm.wxs(309) : error LGHT0094 : Unresolved reference to symbol 'WixComponentGroup:Cg.x86_64_Bin' in section 'Product:*'.

Thank you!
P.S.: The reason I need to build it is because I want to contribute a little improvement to the memory dump program to be able to dump different pages of EEPROM

**Comment by podonoghue** [comment]
Hi gagnonppl,
I'm sorry but the Wix directory is actually to build the installer for entire USBDM package including plugins for Eclipse etc.  Some parts of this installation involve files from other repositories.
The contents of this folder are copied to a parent location along with a lot of other files from the other repositories.  It is rather a mess.  I have not spent much time making this portable as  you need a signing license to make an installer in any case.
I will add a build folder with some information about this but probably not enough to be useful.

You should be able to dump EEPROM by modifying the Page address register entry in the memory dump program or using flat memory model (depending on what target you are using).
bye

**Comment by gagnonpl** [comment]
Hi @podonoghue, thanks for the reply.  I understand for the installer; no matter I just ran the program directly.

About the EEPROM, the problem is that the window area for flash pages in local address space is different from the window area for eeprom pages on the HCS12:
> EEPROM window: 0x0800-0x0C00
> Flash window: 0x8000-0xC000

In addition, there's a different register called EPAGE used to set which page of EEPROM is accessible in the EEPROM window, .  For example, on my device:
> PPAGE: 0x15  (select flash page)
> EPAGE: 0x17 (select EEPROM page)

I have modified the MemoryDump program to add this functionality.  I will open a merge request shortly.
